File: blink.py
from time import sleep
import RPi.GPIO as GPIO
GPIO.setmode(GPIO.BCM)
GPIO.setup(4, GPIO.OUT)
GPIO.setup(17, GPIO.OUT)
try:
	while 1:
		GPIO.output(4, False)
		sleep(1)
		GPIO.output(4, True)
		sleep(1)
		
		GPIO.output(17, False)
		sleep(1)
		GPIO.output(17, True)
		sleep(1)
		
		
except:
	GPIO.cleanup()

File: blink.rb
require 'wiringpi'

pin_1 = 7
pin_2 = 0

io1 = WiringPi::GPIO.new
io2 = WiringPi::GPIO.new

io1.mode(pin_1,OUTPUT)
io2.mode(pin_2,OUTPUT)

trap('INT') {@interrupt = true }

until @interrupt do
	io1.write(pin_1,HIGH)
	sleep 1
	io1.write(pin_1,LOW)
	io2.write(pin_2,HIGH)
	sleep 1 
	io2.write(pin_2,LOW)
end

exit
